tests/unit/forklift/test_legacy.py

```diff
@@ -2000,11 +2000,18 @@ def test_upload_fails_with_diff_filename_same_blake2(
             ("no-way-{version}.tar.gz", "sdist", "no"),
             ("no_way-{version}-py3-none-any.whl", "bdist_wheel", "no"),
             # multiple delimiters
-            ("foo__bar-{version}-py3-none-any.whl", "bdist_wheel", "foo-.bar"),
+            ("foobar-{version}-py3-none-any.whl", "bdist_wheel", "foo-.bar"),
```

Comment:
This was an invalid wheel filename to begin with, the key here is that the project name doesn't match and the suggestion needs to be normalized.

Reply:
The comment above this example is for multiple delimiters, as denoted by `__` - is that covered by another test case somewhere?

Reply:
The multiple delimiters are in `foo-.bar` and are still covered by this. The resulting filename doesn't really matter as long as it doesn't match.